rest_framework/permissions.py

```diff
@@ -175,7 +175,7 @@ class DjangoModelPermissions(BasePermission):
     # Override this if you need to also provide 'view' permissions,
     # or if you want to provide custom permission codes.
     perms_map = {
-        'GET': [],
+        'GET': ['%(app_label)s.view_%(model_name)s'],
```

Comment:
This is new in 2.1 and will likely not be added until the oldest Django we support is 2.1 or add some condition upon Django version here.

Reply:
Ok, my fault.

Reply:
not your fault. DRF has a requirement to support multiple Django versions.